openssl/src/crypto/pkey.rs

```diff
@@ -69,6 +70,22 @@ impl PKey {
         }
     }
 
+    /// Reads private key from PEM, takes ownership of handle
```

Comment:
Could this be moved to a method on `PKey`? Something like `PKey::from_pem` maybe.